keps/sig-auth/0000-20170814-bounding-self-labeling-kubelets.md

```diff
@@ -105,8 +105,35 @@ and warns when invoked with `kubernetes.io` or `k8s.io` labels outside that set.
 * NodeRestriction admission prevents kubelets from adding/removing/modifying `kubernetes.io` or `k8s.io`
 labels other than the specifically allowed labels/prefixes described above on Node *update* only
 
-v1.15:
-
+v1.14:
+
+* Begin migration/removal of in-tree `--node-labels` use outside of the allowed set by addons:
+  * `beta.kubernetes.io/fluentd-ds-ready`
+    * addon: remove from the nodeSelector
+    * kube-up: remove from the default `--node-labels` flag
+  * `beta.kubernetes.io/metadata-proxy-ready`
+    * addon: announce the nodeSelector will switch to `cloud.google.com/metadata-proxy-ready` in 1.15
+    * kube-up: add `cloud.google.com/metadata-proxy-ready=true` along with the existing label to `--node-labels`
+    * kube-up: add `cloud.google.com/metadata-proxy-ready=true` to existing nodes with the `beta.kubernetes.io/metadata-proxy-ready=true` label
+  * `beta.kubernetes.io/kube-proxy-ds-ready`
+    * addon: announce the nodeSelector will switch to `node.kubernetes.io/kube-proxy-ds-ready` in 1.15
+    * kube-up: add `node.kubernetes.io/kube-proxy-ds-ready=true` along with the existing label to `--node-labels`
+    * kube-up: add `node.kubernetes.io/kube-proxy-ds-ready=true` to existing nodes with the `beta.kubernetes.io/kube-proxy-ds-ready=true` label
+  * `beta.kubernetes.io/masq-agent-ds-ready`
+    * addon: announce the nodeSelector will switch to `node.kubernetes.io/masq-agent-ds-ready` in 1.16
```

Comment:
Is masq agent GCP specific? This is subsumed by https://github.com/GoogleCloudPlatform/netd so I'd again be in favor of using the node annotator during the duration of the deprecation, vs introducing new labels.